### ISSUE TITLE
There is no Android_Emulator_Hypervisor_Driver where it is supposed to be

### ISSUE THREAD

**Issue by SantiSnow** [google/android-emulator-hypervisor-driver-for-amd-processors#44]
Unable to run Android Emulator Hypervisor Driver for AMD Processors installer: Cannot start process, the working directory 'C:\Users\Santiago\AppData\Local\Android\Sdk\extras\google\Android_Emulator_Hypervisor_Driver' does not exist.

I have Hyper-V dissabled and BIOS Virtualization enabled. It has nothing to do with it.

**Comment by SantiSnow** [comment]
Already did this and didnt work:
-Activate Virtualization in bios
-Desactivate Hyper-V and hypervisor platform in "activate/desactivate windows function"
& run powershell as admin and copy :"Disable-WindowsOptionalFeature -Online -FeatureName Microsoft-Hyper-V". Finally, reboot.

**Comment by SikoSoft** [comment]
Did you find a solution?

I encounter the same issue when trying to "install" the driver again any time I try to run my project in Android Studio.

(fresh installation of Android Studio, from today)

**Comment by SantiSnow** [comment]
No, I tried in 2 AMD computers. I formated both of them and tried with a fresh W10 install, and then tried to install it following the instructions and taking the considerations above. But nothing, and I am only using it from work in a Intel PC.

**Comment by Gianguyen1234** [comment]
You can try this way, I fix it by turning on virtuallization on Bios:
https://youtu.be/9LVazV4IJ_g
